2024-03-21T02:01:08.718Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 38, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171818d85746a5288008701d8ff26e21320830244ef9498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000000583e9793b960cad7f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb98ab0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27c001a078549c30f6db6447ab35aa2154df946a5d538e932dc5ac2a315bcac1dccbce00a0165add72da73ce249d7c6d0ddb3cae96da4a9d947171e2a5d7edb52f3a7fffa6" ] }, code=UNKNOWN_ERROR, version=6.11.1)
2024-03-21T02:01:08.891Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 39, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171818d85746a5288008701d8ff26e21320830244ef9498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000000583e9793b960cad7f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb98a10000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27c001a071f531a4026cf086a0a804d09e3ca7ea77f6927701a21af89919d5ae43969bf8a0677824f0a1d64894f803b8faf149e9fee4a27a46d18c88a1d1f2164c61cb20cf" ] }, code=UNKNOWN_ERROR, version=6.11.1)
2024-03-21T02:01:10.071Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 34, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171818d85746a5288008701d8ff26e21320830244ef9498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000000583e9793b960cad7f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb98b50000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27c080a0339218135568e63f60e7d7fbabab4ba15c4de00b556acc62eb540731090f68cfa0398b1a8013d8bb99546e93196db690dd7309471cd7c213bf7dd3a155cb748664" ] }, code=UNKNOWN_ERROR, version=6.11.1)
2024-03-21T02:01:10.410Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 35, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171818d85746a5288008701d8ff26e21320830244ef9498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000000583e9793b960cad7f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb98b50000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27c080a0339218135568e63f60e7d7fbabab4ba15c4de00b556acc62eb540731090f68cfa0398b1a8013d8bb99546e93196db690dd7309471cd7c213bf7dd3a155cb748664" ] }, code=UNKNOWN_ERROR, version=6.11.1)
2024-03-21T02:01:20.571Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 36, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90177820171818f85746a5288008701ca25c1e3aea28302768a9498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000000583e9793b960cad7f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb98b50000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27c0019f727a808833f95b2a2ac0ffe5e1d32cf8ac7758d25acbd942265dd5d5942f01a01aa64643cb07a84a89962887fb381429c6337bcabe0a1e9ec98469e5b47387aa" ] }, code=UNKNOWN_ERROR, version=6.11.1)
2024-03-21T02:01:23.592Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 12, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f8b6820171819085746a5288008701ca25c1e3aea282945994efd766ccb38eaf1dfd701853bfce31359239f30580b844095ea7b300000000000000000000000098bf93ebf5c380c0e6ae8e192a7e2ae08edacc0200000000000000000000000000000000000000000000000583e9793b960cad7fc001a0b7e2891f9a71f397d8fdb06bcf428fca990d0747910f143340d2a42fd6f01af8a022e71d9398e8c4519179c970a7a276467492b1f053b9de48f1e1887fd408f638" ] }, code=UNKNOWN_ERROR, version=6.11.1)
2024-03-21T02:01:32.773Z - An error occurred: Error: execution reverted: "TransferHelper::transferFrom: transferFrom failed" (action="estimateGas", data="0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000315472616e7366657248656c7065723a3a7472616e7366657246726f6d3a207472616e7366657246726f6d206661696c6564000000000000000000000000000000", reason="TransferHelper::transferFrom: transferFrom failed", transaction={ "data": "0x38ed173900000000000000000000000000000000000000000000000583e9793b960cad7f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb98c90000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "to": "0x98bf93ebf5c380C0e6Ae8e192A7e2AE08edAcc02" }, invocation=null, revert={ "args": [ "TransferHelper::transferFrom: transferFrom failed" ], "name": "Error", "signature": "Error(string)" }, code=CALL_EXCEPTION, version=6.11.1)
2024-03-21T02:01:35.504Z - Executed DAI_TO_PLS trade at price: 0.000127438. Trade is now active.
2024-03-21T02:01:35.798Z - Executed DAI_TO_PLS trade at price: 0.0001274389. Trade is now active.
2024-03-21T02:18:38.392Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 35, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171819685746a5288008703e0e9a736af3e83023f299498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000a8907bcc34d849126a1a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb9cc50000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305c001a0c983fd17fe2478a4ff1f8371b263f41198a619dd259aade877abe817976f1a1fa048d9832cb24bfaefc19f34caccb17e241a304691c2eb09786fcc3bb07057b0ca" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-21T02:18:42.758Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 39, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171819785746a5288008703e0e9a736af3e83023f299498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000a8907bcc34d849126a1a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb9cbb0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305c080a02a7d94ad395f3c00c45a6e457f9c4b418a40c19658634a7499a78cb141e0709ba017552b8cf6f2f1dae1e35c470467f17ca7fb7e40ebccb4ba2836c1f8fcb24928" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-21T02:18:53.477Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 37, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171819985746a52880087038f417d07ad5a830270c39498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000a8907bcc34d849126a1a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb9cc50000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305c001a0b4330061dd81ce2ac1b0d7551cb70d068dbd353fca6ee0c1b2eab4a8843bc019a0145456cc17eff0d36168837fd625a1e443840f98b94a99391e36ecd0366f8ec0" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-21T02:18:53.614Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 34, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171819985746a52880087038f417d07ad5a830270c39498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000a8907bcc34d849126a1a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb9cd90000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305c080a009cba1242e9ef07702cab2f45015a8254ece12f9b25b5d80c4b3a61dc1cbf90ba062ab2107340da4a7dc1b27201b1899751c3dd3ea5e29d78d874b557e685aabc4" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-21T02:19:16.957Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "ALREADY_EXISTS: already known" }, payload={ "id": 41, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171819c85746a528800870366c105505c42830270c39498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000a8907bcc34d849126a1a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb9ce30000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305c001a0d587f6c924d51fe58da5407a4f3d6afbac3fb007ebd1dc31ab1c4f754b80c969a07ea59a3bb58b2d8e1e99bdf844380e701aa247f6b0822bac953ec87d4efd8962" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-21T02:19:17.622Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "ALREADY_EXISTS: already known" }, payload={ "id": 38, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f90178820171819c85746a528800870366c105505c42830270c39498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000a8907bcc34d849126a1a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb9ce30000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305c001a0d587f6c924d51fe58da5407a4f3d6afbac3fb007ebd1dc31ab1c4f754b80c969a07ea59a3bb58b2d8e1e99bdf844380e701aa247f6b0822bac953ec87d4efd8962" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-21T02:19:34.130Z - Executed PLS_TO_DAI trade at price: 0.0001286803. Trade is now active.
2024-03-21T02:19:34.896Z - An error occurred: Error: execution reverted: "TransferHelper::transferFrom: transferFrom failed" (action="estimateGas", data="0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000315472616e7366657248656c7065723a3a7472616e7366657246726f6d3a207472616e7366657246726f6d206661696c6564000000000000000000000000000000", reason="TransferHelper::transferFrom: transferFrom failed", transaction={ "data": "0x38ed173900000000000000000000000000000000000000000000a8907bcc34d849126a1a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb9cf70000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "to": "0x98bf93ebf5c380C0e6Ae8e192A7e2AE08edAcc02" }, invocation=null, revert={ "args": [ "TransferHelper::transferFrom: transferFrom failed" ], "name": "Error", "signature": "Error(string)" }, code=CALL_EXCEPTION, version=6.9.2)
2024-03-21T02:19:34.898Z - Executed PLS_TO_DAI trade at price: 0.0001286803. Trade is now active.
2024-03-21T02:19:35.959Z - An error occurred: Error: execution reverted: "TransferHelper::transferFrom: transferFrom failed" (action="estimateGas", data="0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000315472616e7366657248656c7065723a3a7472616e7366657246726f6d3a207472616e7366657246726f6d206661696c6564000000000000000000000000000000", reason="TransferHelper::transferFrom: transferFrom failed", transaction={ "data": "0x38ed173900000000000000000000000000000000000000000000a8907bcc34d849126a1a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fb9cf70000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "to": "0x98bf93ebf5c380C0e6Ae8e192A7e2AE08edAcc02" }, invocation=null, revert={ "args": [ "TransferHelper::transferFrom: transferFrom failed" ], "name": "Error", "signature": "Error(string)" }, code=CALL_EXCEPTION, version=6.9.2)
2024-03-21T02:19:36.225Z - An error occurred: Error: transaction execution reverted (action="sendTransaction", data=null, reason=null, invocation=null, revert=null, transaction={ "data": "", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "to": "0xA1077a294dDE1B09bB078844df40758a5D0f9a27" }, receipt={ "_type": "TransactionReceipt", "blockHash": "0x5c7d92a80ea37c6b78d20420c860180cd6ee60c6a08c54a9ea56ccbb190600ba", "blockNumber": 19908614, "contractAddress": null, "cumulativeGasUsed": "17528744", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "gasPrice": "491373820695293", "gasUsed": "31414", "hash": "0x04199338c5ee9557e8c74477a2dfca776ec859a6d83f8bd2adbd1e8504c7cfa6", "index": 78, "logs": [  ], "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000", "root": null, "status": 0, "to": "0xA1077a294dDE1B09bB078844df40758a5D0f9a27" }, code=CALL_EXCEPTION, version=6.9.2)
2024-03-22T15:19:07.712Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "ALREADY_EXISTS: already known" }, payload={ "id": 12, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f8b682017181a085746a5288008701f10b872e7e0882a1ae94efd766ccb38eaf1dfd701853bfce31359239f30580b844095ea7b300000000000000000000000098bf93ebf5c380c0e6ae8e192a7e2ae08edacc0200000000000000000000000000000000000000000000000589685555ee792ce1c001a05b06a23ca9b94a7f92d2a91e911e19a9c60db71a604bd12e0fc44a0e38879400a0553f4840e4c0206a4402ee1a2573d774d20014ed5f46fadeb4e1667c6d7c6a31" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-22T15:19:23.782Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 38, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f9017882017181a385746a5288008701eb6a32f855048302768a9498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000000589685555ee792ce1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fda53d0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27c001a097c941716a9368627ce97913a5f554c9db0440eeef28b8dedcde6270d4241f98a048cbd7b78deb2305fbc522279c41075ad544bb29c74059e99f8f456481401beb" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-22T15:19:38.452Z - An error occurred: Error: could not coalesce error (error={ "code": -32000, "message": "INTERNAL_ERROR: could not replace existing tx" }, payload={ "id": 35, "jsonrpc": "2.0", "method": "eth_sendRawTransaction", "params": [ "0x02f9017882017181a785746a5288008701d0e1ba90d9568302768a9498bf93ebf5c380c0e6ae8e192a7e2ae08edacc0280b9010438ed173900000000000000000000000000000000000000000000000589685555ee792ce1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fda5510000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27c080a0606f0c2ef306d7cc8dfdd58189d2ea118cc98e93fe858622e5f6df1874c241ffa05cf05a112a7cdd7d38a8e02861c379292281e95e490cec9ce901e26f451614ec" ] }, code=UNKNOWN_ERROR, version=6.9.2)
2024-03-22T15:19:46.040Z - An error occurred: Error: transaction execution reverted (action="sendTransaction", data=null, reason=null, invocation=null, revert=null, transaction={ "data": "", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "to": "0xefD766cCb38EaF1dfd701853BFCe31359239F305" }, receipt={ "_type": "TransactionReceipt", "blockHash": "0x106963a0bbdd37775b629cd2da1692f6747e82907a91684baf6467c5840340fd", "blockNumber": 19921728, "contractAddress": null, "cumulativeGasUsed": "23350375", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "gasPrice": "282783965715175", "gasUsed": "37853", "hash": "0x9f175597cb6f47175bfbd67ff8c431797ba004cfd4e5c50cbd044ca8dd330817", "index": 109, "logs": [  ], "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000", "root": null, "status": 0, "to": "0xefD766cCb38EaF1dfd701853BFCe31359239F305" }, code=CALL_EXCEPTION, version=6.9.2)
2024-03-22T15:19:46.320Z - An error occurred: Error: transaction execution reverted (action="sendTransaction", data=null, reason=null, invocation=null, revert=null, transaction={ "data": "", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "to": "0xefD766cCb38EaF1dfd701853BFCe31359239F305" }, receipt={ "_type": "TransactionReceipt", "blockHash": "0x106963a0bbdd37775b629cd2da1692f6747e82907a91684baf6467c5840340fd", "blockNumber": 19921728, "contractAddress": null, "cumulativeGasUsed": "23388228", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "gasPrice": "282783965715175", "gasUsed": "37853", "hash": "0x6813cd49d61d40c5b62733099af4b7e393763188352aba7e255b28792d6d9d78", "index": 110, "logs": [  ], "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000", "root": null, "status": 0, "to": "0xefD766cCb38EaF1dfd701853BFCe31359239F305" }, code=CALL_EXCEPTION, version=6.9.2)
2024-03-22T15:19:47.889Z - Stop-loss triggered: Closing PLS_TO_DAI trade. Resetting trade status.
2024-03-22T15:19:48.255Z - Stop-loss triggered: Closing PLS_TO_DAI trade. Resetting trade status.
2024-03-22T15:19:49.060Z - Stop-loss triggered: Closing PLS_TO_DAI trade. Resetting trade status.
2024-03-22T15:19:50.513Z - An error occurred: Error: execution reverted: "TransferHelper::transferFrom: transferFrom failed" (action="estimateGas", data="0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000315472616e7366657248656c7065723a3a7472616e7366657246726f6d3a207472616e7366657246726f6d206661696c6564000000000000000000000000000000", reason="TransferHelper::transferFrom: transferFrom failed", transaction={ "data": "0x38ed173900000000000000000000000000000000000000000000000589685555ee792ce1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000004025c90c51a811ec329837ebcb6434789ed328b90000000000000000000000000000000000000000000000000000000065fda55b0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000efd766ccb38eaf1dfd701853bfce31359239f305000000000000000000000000a1077a294dde1b09bb078844df40758a5d0f9a27", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "to": "0x98bf93ebf5c380C0e6Ae8e192A7e2AE08edAcc02" }, invocation=null, revert={ "args": [ "TransferHelper::transferFrom: transferFrom failed" ], "name": "Error", "signature": "Error(string)" }, code=CALL_EXCEPTION, version=6.9.2)
2024-03-22T15:19:51.048Z - An error occurred: Error: transaction execution reverted (action="sendTransaction", data=null, reason=null, invocation=null, revert=null, transaction={ "data": "", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "to": "0xefD766cCb38EaF1dfd701853BFCe31359239F305" }, receipt={ "_type": "TransactionReceipt", "blockHash": "0xdc0d6a110210dd8ef081e32f5bcd0c45333c1a13a40f767e6623161ed5860226", "blockNumber": 19921729, "contractAddress": null, "cumulativeGasUsed": "5414489", "from": "0x4025C90C51a811EC329837EBCB6434789ed328B9", "gasPrice": "302565550952200", "gasUsed": "37853", "hash": "0x583f0348cff4899fac522817c880092af9093a5faadbe38d1692593ff57372d5", "index": 31, "logs": [  ], "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000", "root": null, "status": 0, "to": "0xefD766cCb38EaF1dfd701853BFCe31359239F305" }, code=CALL_EXCEPTION, version=6.9.2)
